colossalai/fx/passes/algorithms/ckpt_solver_rotor.py

```diff
@@ -381,7 +326,7 @@ def solver_rotor(gm: ColoGraphModule,
                  mem_limit: int,
                  mem_slots: int = 500,
                  cnode: List[str] = None,
-                 eps: float = 0.02) -> ColoGraphModule:
+                 eps: float = 0.0) -> ColoGraphModule:
```

Comment:
should eps be a very small but non-zero value? e.g. `1e-6`

Reply:
so `0.0` means no memory decay?

Reply:
Yes, the default setting is 0.0

Reply:
ok

Reply:
And the memory decay is calculated by $M(1 - \epsilon)$, maybe the variable name is not that appropriate?

Reply:
Actually the eps will be something around 0.05 or less, `1e-6` is too small as the memory will be discretized.

Reply:
So actually decay is unnecessary if i can estimate the memory accurately.
This can be removed in future if I tested performance of all models

Reply:
I think eps is ok, you can just provide the equation for memory decay in line 338 to explain how eps affect memory decay.

Reply:
I think this option could be provided for user as we might not be able to catch up with all the models in reality, so there might be some cases our meta info provides bad estimations. With this option the user might be able to tune the solver if necessary.